src/uv.cc

```diff
@@ -59,6 +59,16 @@ using v8::ReadOnly;
 using v8::String;
 using v8::Value;
 
+void GetErrMessage(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  int err;
```

Comment:
This can be simplified as:

```suggestion
  int err = args[0].As<Int32>()->Value()
```

Reply:
That is true, fixing it